content/en/docs/concepts/storage/storage-limits.md

```diff
@@ -47,10 +52,9 @@ The limit applies to the entire cluster, so it affects all Nodes.
 
 ## Dynamic volume limits
 
-{{< feature-state state="beta" for_k8s_version="v1.12" >}}
+{{< feature-state state="stable" for_k8s_version="v1.17" >}}
 
-Kubernetes 1.11 introduced support for dynamic volume limits based on Node type as an Alpha feature.
-In Kubernetes 1.12 this feature is graduating to Beta and will be enabled by default.
+Support for dynamic volume limits in Kubernetes 1.17 has been declared stable.
```

Comment:
You can omit this line: the feature-state shortcode says the same thing.

Reply:
Removed